config/locales/interview.yml

```diff
@@ -49,14 +49,20 @@ en:
         provider_interface/interview_wizard:
           attributes:
             date:
-              invalid: The interview date must be a real date
-              missing_values: The interview date must include a %{missing_details}
-              blank: Enter the interview date
-              past: The interview date must be in the future
-              after_rdb: The interview date must be before the application closing date
+              invalid: Date must be a real date
```

Comment:
`invalid`, `missing_values` and `blank` can be removed (as they no longer map to the date errors anyway, but should also default to the same errors used across the project which more or less maps the the amended error messages)